apiserver/Makefile

```diff
@@ -66,9 +66,6 @@ CONTAINER_MARKER=$(CONTAINER_CREATED)
 BINDIR=bin
 endif
 
-# Use ARCH specific build image.
-CALICO_BUILD:=$(CALICO_BUILD)-$(ARCH)
```

Comment:
Doesn't look like we need a architecture specific go-build to build apiserver for different architectures.